frontend/tailwind.config.js

```diff
@@ -1,10 +1,12 @@
 /** @type {import('tailwindcss').Config} */
 const defaultTheme = require('tailwindcss/defaultTheme');
+
 export default {
     content: ['./index.html', './src/**/*.{js,ts,jsx,tsx}'],
     theme: {
         colors: {
             transparent: 'transparent',
+            'white': '#ffffff',
```

Comment:
You should add `lm`- prefix before white

Reply:
ok, but what does this prefix mean?

Reply:
![image](https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-lime/assets/72942456/60865120-22ce-4937-9e80-a3728c2805a5)

Reply:
Ok, thanks